packages/api-proxy/src/v3/eth/wallets/dto/get-deposit-addresses-option.dto.ts

```diff
@@ -1,4 +1,4 @@
-class GetDepositAddressOption {
+export class GetDepositAddressOption {
```

Comment:
If we does not use `export`, we can use the type without `import`.
I don't know the reason but it seems awkward. We should use `import`statement to use a class in another file.

Reply:
`GetDepositAddressOption` is used in the `packages/api-proxy/src/v3/eth/wallets/wallets.service.ts:133`